src/handlebars/installation.handlebars

```diff
@@ -334,9 +334,12 @@
       <pre><code class="highlighted">sudo apt-get install -y software-properties-common</code></pre>
 
       <p>Then repeat the first command.</p>
-      <p>3. Refresh your package list with <tt>sudo apt-get update</tt> and then install your chosen AdoptOpenJDK package. For example, to
+      <p>3. Refresh your package list with and then install your chosen AdoptOpenJDK package. For example, to
```

Comment:
Grammar doesn't make sense here?  Refresh your package list with and...

Reply:
Oops, yeah my PR should have deleted the word `with` as well.